tracer/test/Datadog.Trace.Tests/Configuration/ConfigurationSourceTests.cs

```diff
@@ -180,19 +180,22 @@ public void DefaultSetting(Func<TracerSettings, object> settingGetter, object ex
 
                 // after load settings we can restore the original DD_SERVICE
                 Environment.SetEnvironmentVariable(ConfigurationKeys.ServiceName, originalServiceName, EnvironmentVariableTarget.Process);
+
+                // restore original value
+                Environment.SetEnvironmentVariable(key, originalValue, EnvironmentVariableTarget.Process);
             }
             else
             {
                 Environment.SetEnvironmentVariable(key, value, EnvironmentVariableTarget.Process);
                 IConfigurationSource source = new EnvironmentConfigurationSource();
                 settings = new TracerSettings(source);
+
+                // restore original value
+                Environment.SetEnvironmentVariable(key, originalValue, EnvironmentVariableTarget.Process);
```

Comment:
Both branches call this, can we move it below the `if..else..` to make that a bit more obvious?

Reply:
you're right, done in f55b55d